changelog/unreleased/revisions-impersonate

```diff
@@ -0,0 +1,5 @@
+Bugfix: impersonate owner on Revisions
```

Comment:
Maybe it's worth to have a single changelog entry. Or at least don't change the previous entry.

Reply:
Oops, yeah I tried just changing it first but the changelog checker didn't like that